index.html

```diff
@@ -31,12 +31,133 @@ <h1 class="about-title">I’m Martin.Glad to see you!</h1>
         </p>
         <p class="connection">LET’S CONNECT</p>
         <ul>
-            <li><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
+            <li ><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
             <li><a href="#"><img src="image/Linkedin icon.png" alt="linkedin"></a></li>
             <li><a href="#"><img src="image/Vector (11).png" alt="mail"></a></li>
             <li><a href="#"><img src="image/Vector (12).png" alt="github"></a></li>
             <li><a href="#"><img src="image/Vector (13).png" alt="peace"></a></li>
         </ul>
     </section>
+    <!-- WORK SECTION -->
+    <section class="work">
+        <!-- CARD 1 -->
+        <div class="card-1">
+            <img src="image/Snapshoot Portfolio.png" alt="image1" >
+            <h1 class="tonic">Tonic</h1>
+            <div class="canop">
+                <p class="can-info">CANOPY</p>
+                <div class="point"></div>
+                <p class="back">Back End Dev</p>
+                <div class="point"></div>
+                <p class="back">2015</p>
+            </div>
```

Comment:
- [ ] As mentioned before, I would recommend using proper font styles for the title to match the design. Additionally, the current margin-top between the title and the surrounding `div` does not match the desired gap in the design. To address this, I recommend using flexbox to create a wrapper around the elements and applying the appropriate spacing using the `gap` property. This will ensure consistent and accurate spacing between the title and the surrounding `div`. Please make these adjustments to align with the design specifications. Keep up the good work! 👍